src/libmeasurement_kit/common/libevent_reactor.hpp

```diff
@@ -101,12 +101,18 @@ class LibeventReactor : public Reactor, public NonCopyable, public NonMovable {
 
```

Comment:
This is actually a bugfix for a bug noticed while working on this PR. See 0477c8c79bf1e8507a84fd35bfd42ef29369e8bd.

(Can become a separate PR, if needed.)

Reply:
Is now a separate PR #1646